client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
User Details page
![Screenshot of the User Details page showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/81966c6c-3142-4b05-9815-e8ec5a7c7bca)

Reply:
This now appears as 16px.